ts/Extensions/RangeSelector.ts

```diff
@@ -1318,6 +1318,7 @@ class RangeSelector {
                 dataMax = dataAxis.dataMax;
 
             value = (options.inputDateParser || defaultInputDateParser)(inputValue, chart.time.useUTC);
+            value += (chart.time.timezoneOffset || 0) * 60 * 1000; // #14416
```

Comment:
In the [defaultInputDateParser](https://github.com/highcharts/highcharts/blob/v8.2.2/ts/Extensions/RangeSelector.ts/#L1282), we are already correction the values for `date.getTimezoneOffset`. I am wondering if we can improve that correction to use the current state of `chart.time`, by using `time.getTimezoneOffset` instead. We may have to pass in the `time` as an argument though, in case `defaultInputDateParser` is not bound to the `RangeSelector` scope.

Reply:
The `date.getTimezoneOffset` correction in `defaultInputDateParser` seems to just be correcting for the #433 fix that happened before `defaultInputDateParser` existed :)

Reply:
Okay, but can we combine them? Using `time.getTimezoneOffset` should in theory apply to the old issue as well.